SKILL.tsv

```diff
@@ -5734,302 +5734,302 @@ SKILL_20151102_005733	$[Merkabah] petrifies [Cursed] enemies for 3 seconds.
+SKILL_20151224_005767	Summon Guild Member
+SKILL_20151224_005768	Summon designated guild member to their current location. Need an approval from the designated guild member. Can't summon guild members in an instance dungeon.
+SKILL_20151224_005769	Guild member summon count: #{CaptionRatio}#
+SKILL_20151224_005770	Warp To Guild Member
+SKILL_20151224_005771	Warp to designated guild member's location. Can't warp to guild members in an instance dungeon.
+SKILL_20151224_005772	Warp To Designated Guild Member's Location
+SKILL_20151224_005773	Build Guild Tower
+SKILL_20151224_005774	Build guild tower in the designated location. Can only be built in a field and can't be built in an instance dungeon.{nl}Level 1: Warp allowed{nl}Level 2: Storage allowed{nl}Level 3: Guild growth allowed{nl}Level 4: Guild event allowed{nl}Level 5: Guild hideout allowed
+SKILL_20151224_005775	Guild Tower Function Improvement
+SKILL_20151224_005776	Reduce Craft Time
+SKILL_20151224_005777	Plant a flag that reduces crafting time of all guild members in the nearby area. Can only be used near the guild tower.
+SKILL_20151224_005778	Decrease craft time: #{CaptionRatio}#% 
+SKILL_20151224_005779	Battle Orders
+SKILL_20151224_005780	Increase the strength of guild members around the guild master. Effect disappears if the guild master is too far away.
+SKILL_20151224_005781	Strength increase: #{CaptionRatio}#{nl}Duration: #{CaptionRatio2}#second
+SKILL_20151224_005782	{#993399}{ol}[Magical]{/}{/}{nl}Attack continuously with magic bullets. New bullets are shot if it successfully targets an enemy.
+SKILL_20151224_005783	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio}#%{nl}Bullet count: #{CaptionRatio2}#
+SKILL_20151224_005784	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio}#%{nl}{#339999}{ol}[Ice]{/}{/} Duration: 5 second
+SKILL_20151224_005785	Ice Wall count: #{CaptionRatio}#{nl}{#339999}{ol}[Ice]{/}{/} Duration: 5 second{nl}Ice Wall duration: 15 second
+SKILL_20151224_005786	Create a curtain of ice to protect themselves. Damage the enemy if the attack is successfully blocked and freeze the enemy with a certain rate.
+SKILL_20151224_005787	Duration: #{CaptionTime}#second{nl}Enemies attacked {#339999}{ol}[Ice]{/}{/}Rate: #{CaptionRatio2}#%{nl}{#339999}{ol}[Ice]{/}{/} Duration: #{CaptionRatio}#second
+SKILL_20151224_005788	{#339999}{ol}[Ice]{/}{/}Rate: #{CaptionRatio}#%{nl}Ice Tree duration: #{CaptionTime}#second
+SKILL_20151224_005789	Evocation
+SKILL_20151224_005790	{#993399}{ol}[Magical] - [Dark]{/}{/}{nl}Summon a devil by using a grimoire equipped with a secondary card to attack enemies.
+SKILL_20151224_005791	{#993399}{ol}[Magical]{/}{/}{nl}Summon and release blood sucking bats against enemies to deal damage.
+SKILL_20151224_005792	Pass
+SKILL_20151224_005793	{#993399}{ol}[Magical]{/}{/}{nl}Decrease the cooltime of you and your party members
```

Comment:
[Magical] -> [Magic]

Decrease a skill's cooldown time of you and your party members.